bot/exts/utils/clean.py

```diff
@@ -15,6 +17,9 @@
 
 log = logging.getLogger(__name__)
 
+# Type alias for checks
+CheckHint = Callable[[Message], bool]
```

Comment:
`Predicate` might be a better name here.

Reply:
I agree, https://github.com/python-discord/bot/pull/1527/commits/477810f86387c85ae6da0b80ffccb40dac60e26e